lib/formatEntry.js

```diff
@@ -11,7 +11,7 @@ warning.create(WARNING_NAME, 'FSTEH002', 'cors attribute invalid.')
 warning.create(WARNING_NAME, 'FSTEH003', 'rel attribute invalid.')
 
 module.exports = function formatEntry (earlyHint, opts) {
```

Comment:
I benchmarked and it seems that using a regex is faster (as it does not generate the intermediate by toLocaleLowerCase):

```js
const linkPrefixRE = /^[lL][iI][nN][kK]:/

module.exports = function formatEntry (earlyHint, opts) {
  if (typeof earlyHint === 'string') return linkPrefixRE.test(earlyHint) ? earlyHint.slice(6) : earlyHint
```

Reply:
before about 8 mio. ops/s in both cases. 66 mio ops/s for true and 84 mio ops for false.

Reply:
I don't mean if you directly push the commit.